src/trait_/common_ord.rs

```diff
@@ -216,3 +240,24 @@ pub fn build_ord_body(trait_: &DeriveTrait, data: &Data) -> TokenStream {
 
 	body
 }
+
+/// Generate a match arm that returns `body` for all incomparable `variants`
+pub fn build_incomparable_pattern(variants: &[Data]) -> Option<proc_macro2::TokenStream> {
```

Comment:
```suggestion
pub fn build_incomparable_pattern(variants: &[Data]) -> Option<TokenStream> {
```
It's already imported.